packages/cli/src/build/rollup.ts

```diff
@@ -85,9 +85,10 @@ export let getRollupConfig = (
         return;
       }
       switch (warning.code) {
+        case "CIRCULAR_DEPENDENCY":
         case "EMPTY_BUNDLE":
         case "EVAL":
-        case "CIRCULAR_DEPENDENCY":
+        case "THIS_IS_UNDEFINED":
```

Comment:
as mentioned in that original issue - throwing for this in source files might still be a good idea, this would complicate thing though, currently, this config factory doesn't have access to the list of all monorepo packages so it cant quite distinguish if this has been reported by Rollup for a dependency of a monorepo package

Reply:
We can make it so we only ignore it when `type === 'umd'` so the errors will happen for the CJS builds(which will always be built)

Reply:
Hm, I guess this would work. I was previously concerned about the same error not being reported for some build and reported for others which could confuse users, but when I think about this now - it makes sense and shouldn't really be confusing.

WDYT about making this actually only error for cjs if it's built at all times? This would avoid duplicating this error when you build both CJS and ESM

Reply:
> WDYT about making this actually only error for cjs if it's built at all times? This would avoid duplicating this error when you build both CJS and ESM

Ok, I see that this is not quite possible because `"module"` is just built together with `"node-dev"` type